src/zenml/analytics/context.py

```diff
@@ -77,7 +77,12 @@ def __enter__(self) -> "AnalyticsContext":
         from zenml.config.global_config import GlobalConfiguration
 
         gc = GlobalConfiguration()
-        self.analytics_opt_in = gc.analytics_opt_in
+        store_info = gc.zen_store.get_store_info()
+
+        if self.in_server:
```

Comment:
Cannot this in init, what's this?

Reply:
My bad again, I was not looking good enough to notice that this is a property of `AnalyticsContext` - I only checked the `__init___` method